packages/feedback/src/SmileField.js

```diff
@@ -3,31 +3,42 @@ import PropTypes from 'prop-types';
 import { useField, useFormikContext } from 'formik';
 import FeedbackButton from './FeedbackButton';
 
-const btnStyles = { flex: 1, margin: '0 2% 0 2%' };
+const btnStyles = {};
+const groupStyles = { display: 'grid', gap: '1rem' };
 
 const SmileField = ({ name, options, onChange, autoFocusFeedbackButton }) => {
   const [{ value }] = useField(name);
   const { setFieldValue } = useFormikContext();
 
-  return options.map((option, i) => (
-    <FeedbackButton
-      autoFocus={i === 0 && autoFocusFeedbackButton}
-      style={btnStyles}
-      key={option.icon}
-      icon={option.icon}
-      iconSize="2x"
-      value={option}
-      active={value && value.icon}
-      onClick={() => {
-        setFieldValue(name, option);
-        if (onChange) {
-          onChange(option);
-        }
+  return (
+    <div
+      style={{
+        ...groupStyles,
+        gridTemplateColumns: `repeat(${options.length}, auto)`,
       }}
+      data-testid="face-options"
     >
-      {option.description}
-    </FeedbackButton>
-  ));
+      {options.map((option, i) => (
+        <FeedbackButton
+          autoFocus={i === 0 && autoFocusFeedbackButton}
+          style={btnStyles}
```

Comment:
Good catch, I hadn't noticed that the object was empty now, I'll remove it. I took out `{ flex: 1, margin: '0 2% 0 2%' because I think percent based margins are a bad pattern, it creates spacing that is inconsistent. Also, margin on both sides of the buttons makes alignment of the edges more difficult. That's why grid gap (see comment above) is the preferred solution now. It's better that the gaps between the children be owned by the parent, and that it only be between the children, not on both sides of each child.